sklearn/preprocessing/tests/test_data.py

```diff
@@ -207,8 +213,9 @@ def test_standard_scaler_dtype(add_sample_weight, sparse_constructor):
         scaler = StandardScaler(with_mean=with_mean)
         X_scaled = scaler.fit(X, sample_weight=sample_weight).transform(X)
         assert X.dtype == X_scaled.dtype
-        assert scaler.mean_.dtype == np.float64
-        assert scaler.scale_.dtype == np.float64
+        # TODO is it a good idea to change this behavior?
```

Comment:
Something to be discussed.

Reply:
Why change this? Always using `float64` is intentional as per #12338.

Reply:
OK, I got it. :)